### PR TITLE
topsql: enable topsql feature by default

### LINKED ISSUE #33194: enable topsql feature by default
## Dev Task

Want to enable Top SQL feature by default.

### PR DESCRIPTION
**Pull request by crazycs520**
Signed-off-by: crazycs520 <crazycs520@gmail.com>

### What problem does this PR solve?

Issue Number: close #33194

### Check List

Tests 

- [x] Unit test

### Release note

```release-note
None
```

### CODE REVIEW

File: session/bootstrap.go [hunk 1]
```diff
@@ -583,11 +583,13 @@ const (
...
 	// version85 updates bindings with status 'using' in mysql.bind_info table to 'enabled' status
 	version85 = 85
+	// version86 changes global variable `tidb_enable_top_sql` value from false to true.
+	version86 = 86
 )
 
 // currentBootstrapVersion is defined as a variable, so we can modify its value for testing.
 // please make sure this is the largest version
-var currentBootstrapVersion int64 = version84
+var currentBootstrapVersion int64 = version86
```
Comment (breeswish) [inline session/bootstrap.go:592]: emm. where is the version85

**Review by breeswish — APPROVED** [review]

**Review by zhongzc — APPROVED** [review]

File: session/bootstrap.go (hunk 1 above)
Comment (crazycs520) [reply session/bootstrap.go:592]: here: https://github.com/pingcap/tidb/pull/33184

**Review by bb7133 — APPROVED** [review]
LGTM

**Pull request merged**